lib/android_pay_card.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Braintree.AndroidPayCard do
+  @moduledoc """
+  AndroidPayCard structs are not created directly, but are built within
+  responsees from other endpoints, such as `Braintree.Customer`.
```

Comment:
Let's add a reference to the official ruby sdk docs here for field descriptions:

```suggestion
  responses from other endpoints, such as `Braintree.Customer`.
  
  For additional reference see:
  https://developers.braintreepayments.com/reference/response/android-pay-card/ruby
```